info.yml

```diff
@@ -29,6 +29,14 @@ configuration:
                      launch commands for available DCCs.
         default_value: false
 
+    register_all_software:
```

Comment:
Because we have multiple levels of filtering in place, I wonder if this should be named differently. Perhaps different description? i.e. we have the high level project/user filtering, but also the product/version filtering as well. I don't have a good suggestion though.